lib/screens/health.ex

```diff
@@ -81,10 +81,20 @@ defmodule Screens.Health do
   defp process_metrics({pid, name, supervisor}) do
     metrics =
       pid
-      |> :recon.info(@process_metrics)
+      |> safe_recon_info(@process_metrics)
       |> Stream.map(fn {metric, value} -> "#{metric}=#{value}" end)
       |> Enum.intersperse(" ")
 
     {name, supervisor, metrics}
   end
+
+  # work around https://github.com/ferd/recon/issues/95
+  # The issue is closed but is still very much an issue.
```

Comment:
Rather than putting an assertion like this in a comment where it can drift apart from the actual status of the issue, I'd prefer updating the issue itself, which I've just done. (someday, presumably, the issue will be closed _and_ fixed)